src/CMakeLists.txt

```diff
@@ -341,6 +371,16 @@ endif(HAVE_GOOD_YASM_ELF64)
 add_library(common_texttable_obj OBJECT
   common/TextTable.cc)
 
+if(LINUX)
```

Comment:
nit, could you move this block down after `set(libcommon_files ..)` and conditional `list(APPEND libcommon_files )` there? so we don't need to introduce `msg_async_event_os_srcs`?